lib/msf/ui/console/driver.rb

```diff
@@ -396,6 +396,8 @@ def on_variable_set(glob, var, val)
       handle_payload(val)
     when 'ssh_ident'
       handle_ssh_ident(val)
+    when 'logtlv'
```

Comment:
Naming things is hard, potentially we should use `tlvlogging` for consistency with `sessionlogging`/`consolelogging` above 🤔

Reply:
:+1: Or even `SessionTlvLogging`.

Reply:
I'd be happy with that :+1:

Potentially we could add these values to setg's tab completion for discoverability too